viewvc-install

```diff
@@ -468,9 +466,7 @@ Just hit [Enter] if a default is okay.
             DESTDIR = default
 
     # Install the files.
-    print(
-        "Installing ViewVC to %s%s:" % (ROOT_DIR, DESTDIR and " (DESTDIR = %s)" % (DESTDIR) or "")
-    )
+    print(f"Installing ViewVC to {ROOT_DIR}{f' (DESTDIR = {DESTDIR})' if DESTDIR else ''}:")
```

Comment:
Although I think it is correctly converted from nested % operator into nested f-string and 'and-or' explession into 'if-else' expression, I prefer spritting it into 2 step operations for readability.